kademlia/src/main/java/ingsw/group1/kademlia/KademliaConstants.java

```diff
@@ -0,0 +1,26 @@
+package ingsw.group1.kademlia;
+
+/**
+ * Class defining constants to be used in a distributed network using the Kademlia protocol.
+ */
+public class KademliaConstants {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'KademliaConstants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4743321)